demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -38,32 +38,46 @@
 #include <stdlib.h>
 #include <string.h>
 
-/* POSIX socket includes. */
-#include <netdb.h>
-#include <poll.h>
-#include <time.h>
+/* POSIX includes. */
 #include <unistd.h>
 
-#include <sys/socket.h>
-#include <sys/types.h>
-
 /* Include Demo Config as the first non-system header. */
 #include "demo_config.h"
 
 /* MQTT API header. */
 #include "mqtt.h"
 
+/* OpenSSL sockets transport implementation. */
+#include "openssl_posix.h"
+
+/* Reconnect parameters. */
+#include "reconnect.h"
+
+/* Clock for timer. */
+#include "clock.h"
 
 /**
- * @brief Size of the network buffer for MQTT packets.
+ * These configuration settings are required to run the mutual auth demo.
+ * Throw compilation error if the below configs are not defined.
  */
-#define NETWORK_BUFFER_SIZE    ( 1024U )
-
-/* Check that client identifier is defined. */
+#ifndef ROOT_CA_CERT_PATH
+    #error "Please define path to root ca certificate of the AWS IoT MQTT broker(ROOT_CA_CERT_PATH) in demo_config.h."
```

Comment:
This file should not mention about Mosquitto broker instead (or I would say just an "_MQTT broker for the demo_" generically as customer may choose a non-Mosquitto broker)

Reply:
Thanks for catching!